docs/StardustDocs/topics/gettingStarted.md

```diff
@@ -1,22 +1,170 @@
+## Create Kotlin project
 
-You can also give us feedback or ask a question in [#datascience](https://kotlinlang.slack.com/archives/C4W52CFEZ) channel of a Kotlin Slack.
+1. In IntelliJ IDEA, select **File** | **New** | **Project**.
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location, if necessary.
 
-Good luck!
+   > Select the **Create Git repository** checkbox to place the new project under version control. You can enable this
+   > later at any time.
+   >
+   {type="tip"}
+
+4. From the **Language** list, select **Kotlin**.
+5. Select the **Gradle** build system.
+6. From the **JDK list**, select the [JDK](https://www.oracle.com/java/technologies/downloads/) that you want to use in
+   your project.
+    * If the JDK is installed on your computer, but not defined in the IDE, select **Add JDK** and specify the path to the
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+7. From the **Gradle DSL** list, select **Kotlin** or **Groovy**.
+8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
+9. Click **Create**.
+
+You have successfully created a project with Gradle.
+
+### Update Gradle dependencies
+
+In your Gradle build file (`build.gradle.kts`), add the DataFrame library as a dependency:
+
+<tabs>
+<tab title="Kotlin DSL">
+
+```kotlin
+dependencies {
+    implementation("org.jetbrains.kotlinx:dataframe:%dataFrameVersion%")
+}
+```
+
+</tab>
+
+<tab title="Groovy DSL">
+
+```kotlin
+dependencies {
+    implementation 'org.jetbrains.kotlinx:dataframe:%dataFrameVersion%'
+}
+```
+
+</tab>
+
+</tabs>
+
+### Add imports
+
+In `src/main/kotlin/Main.kt`, add the following imports at the top of the file:
+
+```kotlin
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.io.*
+import org.jetbrains.kotlinx.dataframe.api.*
+```
+
+## Import data
+
+Download and save our <res resource-id="movie-sample-data"> file to the root directory of your project.
+
+Delete the `println()` functions and comments from your main function in `Main.kt`.
+
+To import the movie sample data into a data frame and print it, inside your main function in `Main.kt`, add the following code:
+
+```kotlin
+    // Import your data to a data frame
+    var df = DataFrame.read("movies.csv")
+
+    // Print your data frame
+    println(df)
+```
+
+## Manipulate data
+
+To print some information about your data frame and sort your data, add the following additional lines of code:
+
+```kotlin
+    // Print some information about the data frame
+    println(df.columnNames()) // Print column names
+    println(df.count())       // Print number of rows
+
+    // Sort your data alphabetically by title and print
+    df = df.sortBy("title")
```

Comment:
Probably let's add the two operations here: selecting just titles and filter it by genres (if it contains Comedy, for example)

Reply:
Updated! If you're happy that this is resolved, please resolve this conversation.